lib/util/glob-util.js

```diff
@@ -10,6 +10,7 @@
 
 const fs = require("fs"),
     path = require("path"),
+    minimatch = require("minimatch"),
```

Comment:
As far as I can tell, glob utils is meant specifically for retrieving a list of matching files from the file system. You give it a glob pattern and path, and it returns a list of matching files. It uses minimatch internally. But in this case we already have a file name and need to determine if it matches a given glob pattern -- glob-utils doesn't expose that functionality.